app/models/pool.rb

```diff
@@ -99,6 +91,18 @@ def search(params)
         q = q.undeleted
       end
 
+      params[:order] ||= params.delete(:sort)
+      case params[:order]
+      when "name"
+        q = q.order("pools.name")
+      when "created_at"
+        q = q.order("pools.created_at desc")
+      when "post_count"
+        q = q.order("pools.post_count desc")
```

Comment:
Removing `order("updated_at desc")` from the controller means it needs to be added here: `order("pools.name, pools.updated_at desc")` and `order("pools.post_count desc, pools.updated_at desc")`.